src/components/Dashboard/Dashboard.tsx

```diff
@@ -0,0 +1,63 @@
+import { useState } from 'react';
+
+import style from './Dashboard.module.scss';
+
+const activities = [
+  { activity: 'running', id: 1, timesDone: 3 },
+  { activity: 'swimming', id: 2, timesDone: 2 },
+  { activity: 'not smoking', id: 3, timesDone: 21 },
+];
+
+export const Dashboard = () => {
+  const [formVisible, setFormVisible] = useState<boolean>(false);
+  const [newActivity, setNewActivity] = useState<string>('');
+  const [currentActivities, setCurrentActivities] = useState([...activities]);
+
+  const handleChange = () => {
+    setNewActivity(event.target.value);
+  };
+
+  const handleSubmit = () => {
+    event.preventDefault();
+    const newObj = { activity: newActivity, id: 4, timesDone: 0 };
+    const newArray = activities.concat([newObj]);
+    setCurrentActivities(newArray);
+    setFormVisible(false);
+  };
+
+  return (
+    <div className={style.container}>
+      <div className={style.box}>
+        <h1 className={style.title}>Všechny aktivity</h1>
+        <ul className={style.activities}>
+          {currentActivities.map((activity) => (
+            <div className={style.activity} key={activity.id}>
+              <li>{activity.activity}</li>
+              <span>{activity.timesDone}</span>
+            </div>
+          ))}
+        </ul>
+        {!formVisible && (
+          <button
+            className={style.revealForm}
+            onClick={() => setFormVisible(true)}
+          >
+            +
+          </button>
+        )}
+        {formVisible && (
+          <form onSubmit={() => handleSubmit()}>
```

Comment:
Usually empty call like this can be shortened to

```suggestion
          <form onSubmit={handleSubmit}>
```

when the passed in (`handleSubmit` in this case) function takes no params, it is then equivalent